tests/unit/Jadu/Pulsar/Twig/Extension/HelperOptionsModifierExtensionTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Jadu\Pulsar\Twig\Extension;
+
+class HelperOptionsModifierExtensionTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        $this->ext = new HelperOptionsModifierExtension();
+    }
+
+    public function testGetName()
+    {
+        $this->assertEquals('modify_options', $this->ext->getName());
+    }
+
+    public function testNullReturnsFalse()
+    {
+        $this->assertEquals(false, $this->ext->modifyOptions(null));
+    }
+
+    public function testOptionsModifiedIfHelpSupplied()
+    {
+        $dataIn = array('help' => 'example help');
+        $dataOut = array('help' => 'example help', 'help_id' => 'guid-1');
```

Comment:
does `$dataOut` need to be in any of these tests? Looks as though for each one it's defined and then unused. Were you planning on using it like ... ?
```
self::assertSame($dataOut['help'], $result['help']);
```